template/code-lstlisting.tex

```diff
@@ -5,13 +5,12 @@
 	\begin{subfigure}{0.48\textwidth}
 		\begin{lstlisting}[numbers=left,xleftmargin=2em]
 			int main() {
-				printf("Hello world!");
+				printf("Hello world!"); (*@\label{line:example:print}@*)
```

Comment:
I changed the preamble recently. Now you only need the @ symbol to escape a latex command.

Reply:
changed